pkg/daemon/ceph/client/pool.go

```diff
@@ -247,6 +248,14 @@ func CreateECPoolForApp(context *clusterd.Context, namespace, poolName, ecProfil
 		}
 	}
 
+	compressionModeProperty := "compression_mode"
```

Comment:
`CreateECPoolForApp()` and `CreateReplicatedPoolForApp()` could use a small improvement while we're in here... They both call `givePoolAppTag()` and now they are both setting the compression mode. How about factoring those out into a helper method for setting pool properties that are common to both EC and replicated pools?

Reply:
Fixed